README.md

```diff
@@ -70,7 +70,7 @@ polytracker --help
 The `polytracker` script is also a REPL, if run with no command line arguments:
 ```python
 $ polytracker
-PolyTracker (3.0.0)
+PolyTracker (3.1.0)
```

Comment:
Suggest that we bump the version to 4.0.0 because this is a non-backwards compatible change with rather large impact on previous behaviour. This should also be reflected in the polytracker.h version.

Reply:
Bumped to 4.0.0